### ISSUE TITLE
Conda build fails

### ISSUE THREAD

**Issue by constantinpape** [svenpeter42/fastfilters#10]
Hi,

I tried to build the conda package because I need a python 3 version (I couldn't find it on the ilastik channel, is there one anywhere else). However, I can't build the package with any python version. I always obtain the following error:

```
Traceback (most recent call last):
  File "/home/consti/software/miniconda/bin/conda-build", line 6, in <module>
    sys.exit(conda_build.cli.main_build.main())
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 387, in main
    execute(sys.argv[1:])
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/cli/main_build.py", line 378, in execute
    noverify=args.no_verify)
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/api.py", line 184, in build
    need_source_download=need_source_download, config=config, variants=variants)
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/build.py", line 1715, in build_tree
    notest=notest,
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/build.py", line 1002, in build
    always_include_files=m.always_include_files())
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda_build/environ.py", line 792, in create_env
    execute_actions(actions, index, verbose=config.debug)
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda/plan.py", line 830, in execute_actions
    execute_instructions(plan, index, verbose)
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda/instructions.py", line 247, in execute_instructions
    cmd(state, arg)
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda/instructions.py", line 108, in UNLINKLINKTRANSACTION_CMD
    txn.execute()
  File "/home/consti/software/miniconda/lib/python2.7/site-packages/conda/core/link.py", line 297, in execute
    rollback_excs,
conda.CondaMultiError: post-link script failed for package defaults::gcc-4.8.5-7
running your command again with `-v` will provide additional information
location of failed script: /home/consti/software/miniconda/conda-bld/fastfilters_1502243349072/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/bin/.gcc-post-link.sh
==> script messages <==
<None>
```

**Comment by constantinpape** [comment]
I am using the latest conda build (3.0.9).

**Comment by stuarteberg** [comment]
This seems to be a complication related to the `gcc` package.  We (ilastik) are abandoning that package dependency in our new builds.  Now I've submitted PR #12 to update the conda recipe.

**Comment by svenpeter42** [comment]
fixed with PR #12 , thanks @stuarteberg  :)
